### ISSUE TITLE
MetaSearch Python error

### ISSUE THREAD

**Issue by gioman** [qgis/QGIS#37748]
QGIS 3.10.7 on Windows.

```
Traceback (most recent call last):
  File "C:/OSGEO4~1/apps/qgis-ltr/./python/plugins\MetaSearch\dialogs\maindialog.py", line 831, in show_metadata
    [self.catalog.records[identifier].identifier])
AttributeError: 'CatalogueServiceWeb' object has no attribute 'records'
```

**Steps to replicate:**

search a catalogue

double click on a result to get the metadata for that particular resource

close the dialog

go to "services" and click on "service info"

go back to to "search" and click on one of the results > error

**Comment by gioman** [comment]
@tomkralidis ping.

**Comment by tomkralidis** [comment]
@gioman I made a fix but need to apply to master, 3.14 branch, 3.12 branch and 3.10 branch. I would like to apply a single fix first for you to test before cherry picking across. Which branch do you prefer?

**Comment by gioman** [comment]
@tomkralidis I guess 3.10. Thanks for taking care of this issue!

**Comment by tomkralidis** [comment]
Fix in #37768
